### DISCUSSION TITLE
Official Action Slack Notification

### DISCUSSION THREAD

**Discussion by pablosproject** [community/community#26040] [Actions]
<p>Hello everyone! Is there an official action that we can drop in our workflows to have notifications on the state of the running action?</p>

**Comment by AndreaGriffiths11** [comment]
<p>Hi <a class="mention" href="/pablosproject">@pablosproject</a>, welcome to the community! While there isn’t an official GitHub action, there are a variety of community developed integrations on the <a href="https://github.com/marketplace?query=slack&amp;type=actions">marketplace</a>.</p>
<p>I hope this helps!</p>

**Comment by pablosproject** [comment, marked as answer]
<p>Yes, i found those, but I wanted something more official to integrate in the action life cycle. Anyway, thanks for the reply, will use what the community provide.</p>

**Comment by acro5piano** [comment]
<p>I think official chat integrations are very helpful. For example, CircleCI officially provides chat notifications for Slack, Hipchat, and so on.</p>
